mesonbuild/modules/python.py

```diff
@@ -752,17 +752,22 @@ def find_installation(self, state: 'ModuleState', args: T.Tuple[T.Optional[str]]
                 else:
                     found_modules.append(mod)
 
-        msg: T.List['mlog.TV_Loggable'] = ['Program', python.name]
-        if want_modules:
-            msg.append('({})'.format(', '.join(want_modules)))
+        msg: T.List['mlog.TV_Loggable'] = ['Runtime dependency', display_name]
```

Comment:
See comment: https://github.com/mesonbuild/meson/pull/10456#issuecomment-1158809399 for the reasoning behind `s/python.name/display_name`

Reply:
Using style from Boost a la https://github.com/mesonbuild/meson/pull/10456#issuecomment-1152874592